website/docs/terraform/terraform-intro.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Introduction to Terraform Controller
+hide_title: true
+---
+
+# Overview
+
+[Terraform Controller](https://github.com/weaveworks/tf-controller) (TF-Controller) is a reliable tool for managing your infrastructure and application resources using the GitOps approach, all at your own pace. An open source project created by Weaveworks, the makers of [Flux](https://fluxcd.io), TF-Controller follows patterns established by Flux and integrates with Weave GitOps. 
```

Comment:
It would be good to update the Overview section in https://weaveworks.github.io/tf-controller/ to remove the word `experimental` from `TF-controller is an experimental controller..`. I feel it contradicts somewhat what we state here i.e. `reliable`.